PyTorch/Segmentation/MaskRCNN/pytorch/tools/train_net.py

```diff
@@ -257,7 +286,10 @@ def main():
 
     if args.skip_checkpoint:
         cfg.SAVE_CHECKPOINT = False
-        
+
+    cfg.SOLVER.IMS_PER_BATCH = num_gpus * 4
+    cfg.SOLVER.BASE_LR = num_gpus / 8 * 0.005
```

Comment:
Can you please describe what is the reason / effect of larger batch size and where is this LR rule coming from?